src/main/java/com/slack/kaldb/logstore/LogStore.java

```diff
@@ -24,6 +24,8 @@
 
   Path getDirectory();
 
-  Collection<String> activeFiles();
+  public IndexCommit getIndexCommit();
   // TODO: Add an isReadOnly and setReadOnly API here.
+
+  public void releaseIndexCommitRef(IndexCommit indexCommit);
```

Comment:
Please move this method before the TODO. 

Also, add a comment on how these 2 methods should be used. We also need to document that the IndexCommit data can vary even if no data is added to index between calls since merge can change the index commit.

Reply:
done